diff --git a/build.gradle b/build.gradle
index edfdf66..c7cfa4d 100644
--- a/build.gradle
+++ b/build.gradle
@@ -40,7 +40,7 @@ dependencies {
     implementation 'com.squareup.okhttp3:okhttp'
     implementation 'com.squareup.okhttp3:okhttp-brotli'
     implementation 'io.sentry:sentry:6.27.0'
-    implementation 'rocks.kavin:reqwest4j:1.0.6'
+    implementation files('libs/reqwest4j.jar')
     implementation 'io.minio:minio:8.5.4'
 }
 
diff --git a/src/main/java/me/kavin/piped/Main.java b/src/main/java/me/kavin/piped/Main.java
index a5a8b57..4f9e6ed 100644
--- a/src/main/java/me/kavin/piped/Main.java
+++ b/src/main/java/me/kavin/piped/Main.java
@@ -41,7 +41,7 @@ public class Main {
         ReqwestUtils.init(Constants.REQWEST_PROXY);
 
         Sentry.init(options -> {
-            options.setDsn(Constants.SENTRY_DSN);
+            options.setDsn("");
             options.setRelease(Constants.VERSION);
             options.addIgnoredExceptionForType(ErrorResponse.class);
             options.setTracesSampleRate(0.1);
