diff --git a/build.gradle b/build.gradle
index f68c2d6..977c514 100644
--- a/build.gradle
+++ b/build.gradle
@@ -40,7 +40,8 @@ dependencies {
     implementation 'com.squareup.okhttp3:okhttp'
     implementation 'com.squareup.okhttp3:okhttp-brotli'
     implementation 'io.sentry:sentry:6.24.0'
-    implementation 'rocks.kavin:reqwest4j:1.0.5'
+    // implementation 'rocks.kavin:reqwest4j:1.0.5'
+    implementation files('libs/reqwest4j.jar')
     implementation 'io.minio:minio:8.5.4'
 }
 
diff --git a/src/main/java/me/kavin/piped/consts/Constants.java b/src/main/java/me/kavin/piped/consts/Constants.java
index 4f338c0..8c606b5 100644
--- a/src/main/java/me/kavin/piped/consts/Constants.java
+++ b/src/main/java/me/kavin/piped/consts/Constants.java
@@ -142,7 +142,7 @@ public class Constants {
             DISABLE_SERVER = Boolean.parseBoolean(getProperty(prop, "DISABLE_SERVER", "false"));
             DISABLE_LBRY = Boolean.parseBoolean(getProperty(prop, "DISABLE_LBRY", "false"));
             SUBSCRIPTIONS_EXPIRY = Integer.parseInt(getProperty(prop, "SUBSCRIPTIONS_EXPIRY", "30"));
-            SENTRY_DSN = getProperty(prop, "SENTRY_DSN", "");
+            SENTRY_DSN = /*getProperty(prop, "SENTRY_DSN", "")*/ "";
             S3_ENDPOINT = getProperty(prop, "S3_ENDPOINT");
             S3_ACCESS_KEY = getProperty(prop, "S3_ACCESS_KEY");
             S3_SECRET_KEY = getProperty(prop, "S3_SECRET_KEY");
