diff --git a/build.gradle b/build.gradle
index 952abb5..582be0d 100644
--- a/build.gradle
+++ b/build.gradle
@@ -11,7 +11,7 @@ repositories {
 }
 
 dependencies {
-    rust(project(":reqwest-jni"))
+    //rust(project(":reqwest-jni"))
 }
 
 // javac -h
diff --git a/reqwest-jni/build.gradle.kts b/reqwest-jni/build.gradle.kts
index 189a81d..8c5895e 100644
--- a/reqwest-jni/build.gradle.kts
+++ b/reqwest-jni/build.gradle.kts
@@ -4,8 +4,10 @@ plugins {
 
 rust {
     release.set(true)
-    command.set("cross")
+    command.set("cargo")
+    cargoInstallTargets.set(true)
 
-    targets += target("aarch64-unknown-linux-gnu", "libreqwest.so")
-    targets += target("x86_64-unknown-linux-gnu", "libreqwest.so")
+    // targets += target("aarch64-unknown-linux-gnu", "libreqwest.so")
+    // targets += target("x86_64-unknown-linux-gnu", "libreqwest.so")
+    targets += target("x86_64-unknown-linux-musl", "libreqwest.so")
 }
